test/compareTimings.m

```diff
@@ -0,0 +1,252 @@
+function compareTimings(statusBefore, statusAfter)
+% COMPARETIMINGS compare timing of matlab2tikz test suite runs
+%
+% This function plots some analysis plots of the timings of different test
+% cases. When the test suite is run repeatedly, the median statistics are
+% reported as well as the individual runs.
+%
+% Usage:
+%  COMPARETIMINGS(statusBefore, statusAfter)
+%
+% Parameters:
+%  - statusBefore and statusAfter are expected to be
+%    N x R cell arrays, each cell contains a status of a test case
+%    where there are N test cases, repeated R times each.
+%
+% A way to build such cells is:
+%
+% suite = @ACID
+% N = numel(suite(0));
+% R = 10;
+% statusBefore = cell(N, R);
+% for r = 1:R
+%     statusBefore(:, r) = testHeadless;
+% end
+% % now check out the after commit
+% statusAfter = cell(N, R);
+% for r = 1:R
+%     statusAfter(:, r) = testHeadless;
+% end
+% compareTimings(statusBefore, statusAfter)
```

Comment:
I would reformat slightly the example and indent it to make it more visible, otherwise else is good:

% Example:
%     suite = @ACID;
%     N = numel(suite(0));
%     R = 10;
%     
%     % Record current timings
%     before = cell(N, R);
%     for r = 1:R
%         before(:, r) = testHeadless;
%     end
%     
%     % now check out the after commit
%     after = cell(N, R);
%     for r = 1:R
%         after(:, r) = testHeadless;
%     end
%         
%     compareTimings(before, after)